[2025-03-14T14:51:46.050+0000] {taskinstance.py:1159} INFO - Dependencies all met for dep_context=non-requeueable deps ti=<TaskInstance: Airflow_ETL_DAG.insert_worldmeter scheduled__2025-03-10T00:00:00+00:00 [queued]>
[2025-03-14T14:51:46.060+0000] {taskinstance.py:1159} INFO - Dependencies all met for dep_context=requeueable deps ti=<TaskInstance: Airflow_ETL_DAG.insert_worldmeter scheduled__2025-03-10T00:00:00+00:00 [queued]>
[2025-03-14T14:51:46.061+0000] {taskinstance.py:1361} INFO - Starting attempt 1 of 2
[2025-03-14T14:51:46.074+0000] {taskinstance.py:1382} INFO - Executing <Task(PythonOperator): insert_worldmeter> on 2025-03-10 00:00:00+00:00
[2025-03-14T14:51:46.081+0000] {standard_task_runner.py:57} INFO - Started process 299 to run task
[2025-03-14T14:51:46.084+0000] {standard_task_runner.py:84} INFO - Running: ['airflow', 'tasks', 'run', 'Airflow_ETL_DAG', 'insert_worldmeter', 'scheduled__2025-03-10T00:00:00+00:00', '--job-id', '17', '--raw', '--subdir', 'DAGS_FOLDER/airflow_etl.py', '--cfg-path', '/tmp/tmp0tq_fgn_']
[2025-03-14T14:51:46.087+0000] {standard_task_runner.py:85} INFO - Job 17: Subtask insert_worldmeter
[2025-03-14T14:51:46.142+0000] {task_command.py:416} INFO - Running <TaskInstance: Airflow_ETL_DAG.insert_worldmeter scheduled__2025-03-10T00:00:00+00:00 [running]> on host 587afbd7d2b9
[2025-03-14T14:51:46.234+0000] {taskinstance.py:1662} INFO - Exporting env vars: AIRFLOW_CTX_DAG_OWNER='airflow' AIRFLOW_CTX_DAG_ID='Airflow_ETL_DAG' AIRFLOW_CTX_TASK_ID='insert_worldmeter' AIRFLOW_CTX_EXECUTION_DATE='2025-03-10T00:00:00+00:00' AIRFLOW_CTX_TRY_NUMBER='1' AIRFLOW_CTX_DAG_RUN_ID='scheduled__2025-03-10T00:00:00+00:00'
[2025-03-14T14:52:00.528+0000] {taskinstance.py:1937} ERROR - Task failed with exception
Traceback (most recent call last):
  File "/home/airflow/.local/lib/python3.8/site-packages/airflow/operators/python.py", line 192, in execute
    return_value = self.execute_callable()
  File "/home/airflow/.local/lib/python3.8/site-packages/airflow/operators/python.py", line 209, in execute_callable
    return self.python_callable(*self.op_args, **self.op_kwargs)
  File "/opt/airflow/dags/airflow_etl.py", line 162, in task_insert_worldmeter
    session.execute(stmt)
  File "/home/airflow/.local/lib/python3.8/site-packages/sqlalchemy/orm/session.py", line 1717, in execute
    result = conn._execute_20(statement, params or {}, execution_options)
  File "/home/airflow/.local/lib/python3.8/site-packages/sqlalchemy/engine/base.py", line 1710, in _execute_20
    return meth(self, args_10style, kwargs_10style, execution_options)
  File "/home/airflow/.local/lib/python3.8/site-packages/sqlalchemy/sql/elements.py", line 334, in _execute_on_connection
    return connection._execute_clauseelement(
  File "/home/airflow/.local/lib/python3.8/site-packages/sqlalchemy/engine/base.py", line 1569, in _execute_clauseelement
    compiled_sql, extracted_params, cache_hit = elem._compile_w_cache(
  File "/home/airflow/.local/lib/python3.8/site-packages/sqlalchemy/sql/elements.py", line 545, in _compile_w_cache
    compiled_sql = self._compiler(
  File "/home/airflow/.local/lib/python3.8/site-packages/sqlalchemy/sql/elements.py", line 567, in _compiler
    return dialect.statement_compiler(dialect, self, **kw)
  File "/home/airflow/.local/lib/python3.8/site-packages/sqlalchemy/sql/compiler.py", line 809, in __init__
    Compiled.__init__(self, dialect, statement, **kwargs)
  File "/home/airflow/.local/lib/python3.8/site-packages/sqlalchemy/sql/compiler.py", line 464, in __init__
    self.string = self.process(self.statement, **compile_kwargs)
  File "/home/airflow/.local/lib/python3.8/site-packages/sqlalchemy/sql/compiler.py", line 499, in process
    return obj._compiler_dispatch(self, **kwargs)
  File "/home/airflow/.local/lib/python3.8/site-packages/sqlalchemy/sql/visitors.py", line 82, in _compiler_dispatch
    return meth(self, **kw)
  File "/home/airflow/.local/lib/python3.8/site-packages/sqlalchemy/sql/compiler.py", line 4017, in visit_insert
    crud_params = crud._get_crud_params(
  File "/home/airflow/.local/lib/python3.8/site-packages/sqlalchemy/sql/crud.py", line 172, in _get_crud_params
    raise exc.CompileError(
sqlalchemy.exc.CompileError: Unconsumed column names: total_cases, new_deaths, new_cases
[2025-03-14T14:52:00.552+0000] {taskinstance.py:1400} INFO - Marking task as UP_FOR_RETRY. dag_id=Airflow_ETL_DAG, task_id=insert_worldmeter, execution_date=20250310T000000, start_date=20250314T145146, end_date=20250314T145200
[2025-03-14T14:52:00.573+0000] {standard_task_runner.py:104} ERROR - Failed to execute job 17 for task insert_worldmeter (Unconsumed column names: total_cases, new_deaths, new_cases; 299)
[2025-03-14T14:52:00.632+0000] {local_task_job_runner.py:228} INFO - Task exited with return code 1
[2025-03-14T14:52:00.650+0000] {taskinstance.py:2778} INFO - 0 downstream tasks scheduled from follow-on schedule check
[2025-03-14T15:24:41.141+0000] {taskinstance.py:1159} INFO - Dependencies all met for dep_context=non-requeueable deps ti=<TaskInstance: Airflow_ETL_DAG.insert_worldmeter scheduled__2025-03-10T00:00:00+00:00 [queued]>
[2025-03-14T15:24:41.157+0000] {taskinstance.py:1159} INFO - Dependencies all met for dep_context=requeueable deps ti=<TaskInstance: Airflow_ETL_DAG.insert_worldmeter scheduled__2025-03-10T00:00:00+00:00 [queued]>
[2025-03-14T15:24:41.158+0000] {taskinstance.py:1361} INFO - Starting attempt 1 of 2
[2025-03-14T15:24:41.173+0000] {taskinstance.py:1382} INFO - Executing <Task(PythonOperator): insert_worldmeter> on 2025-03-10 00:00:00+00:00
[2025-03-14T15:24:41.180+0000] {standard_task_runner.py:57} INFO - Started process 212 to run task
[2025-03-14T15:24:41.183+0000] {standard_task_runner.py:84} INFO - Running: ['airflow', 'tasks', 'run', 'Airflow_ETL_DAG', 'insert_worldmeter', 'scheduled__2025-03-10T00:00:00+00:00', '--job-id', '4', '--raw', '--subdir', 'DAGS_FOLDER/airflow_etl.py', '--cfg-path', '/tmp/tmpu4b44whf']
[2025-03-14T15:24:41.185+0000] {standard_task_runner.py:85} INFO - Job 4: Subtask insert_worldmeter
[2025-03-14T15:24:41.246+0000] {task_command.py:416} INFO - Running <TaskInstance: Airflow_ETL_DAG.insert_worldmeter scheduled__2025-03-10T00:00:00+00:00 [running]> on host 409e4a022234
[2025-03-14T15:24:41.337+0000] {taskinstance.py:1662} INFO - Exporting env vars: AIRFLOW_CTX_DAG_OWNER='airflow' AIRFLOW_CTX_DAG_ID='Airflow_ETL_DAG' AIRFLOW_CTX_TASK_ID='insert_worldmeter' AIRFLOW_CTX_EXECUTION_DATE='2025-03-10T00:00:00+00:00' AIRFLOW_CTX_TRY_NUMBER='1' AIRFLOW_CTX_DAG_RUN_ID='scheduled__2025-03-10T00:00:00+00:00'
[2025-03-14T15:26:33.386+0000] {logging_mixin.py:151} INFO - Insertion des données dans worldmeter terminée.
[2025-03-14T15:26:36.552+0000] {python.py:194} INFO - Done. Returned value was: None
[2025-03-14T15:26:37.052+0000] {taskinstance.py:1400} INFO - Marking task as SUCCESS. dag_id=Airflow_ETL_DAG, task_id=insert_worldmeter, execution_date=20250310T000000, start_date=20250314T152441, end_date=20250314T152637
[2025-03-14T15:26:37.572+0000] {local_task_job_runner.py:228} INFO - Task exited with return code 0
[2025-03-14T15:26:37.811+0000] {taskinstance.py:2778} INFO - 0 downstream tasks scheduled from follow-on schedule check
[2025-03-16T12:05:38.603+0000] {taskinstance.py:1159} INFO - Dependencies all met for dep_context=non-requeueable deps ti=<TaskInstance: Airflow_ETL_DAG.insert_worldmeter scheduled__2025-03-10T00:00:00+00:00 [queued]>
[2025-03-16T12:05:38.615+0000] {taskinstance.py:1159} INFO - Dependencies all met for dep_context=requeueable deps ti=<TaskInstance: Airflow_ETL_DAG.insert_worldmeter scheduled__2025-03-10T00:00:00+00:00 [queued]>
[2025-03-16T12:05:38.616+0000] {taskinstance.py:1361} INFO - Starting attempt 1 of 2
[2025-03-16T12:05:38.632+0000] {taskinstance.py:1382} INFO - Executing <Task(PythonOperator): insert_worldmeter> on 2025-03-10 00:00:00+00:00
[2025-03-16T12:05:38.639+0000] {standard_task_runner.py:57} INFO - Started process 234 to run task
[2025-03-16T12:05:38.643+0000] {standard_task_runner.py:84} INFO - Running: ['airflow', 'tasks', 'run', 'Airflow_ETL_DAG', 'insert_worldmeter', 'scheduled__2025-03-10T00:00:00+00:00', '--job-id', '5', '--raw', '--subdir', 'DAGS_FOLDER/airflow_etl.py', '--cfg-path', '/tmp/tmpal6jb96u']
[2025-03-16T12:05:38.647+0000] {standard_task_runner.py:85} INFO - Job 5: Subtask insert_worldmeter
[2025-03-16T12:05:38.711+0000] {task_command.py:416} INFO - Running <TaskInstance: Airflow_ETL_DAG.insert_worldmeter scheduled__2025-03-10T00:00:00+00:00 [running]> on host 87ae174c8f0e
[2025-03-16T12:05:38.809+0000] {taskinstance.py:1662} INFO - Exporting env vars: AIRFLOW_CTX_DAG_OWNER='airflow' AIRFLOW_CTX_DAG_ID='Airflow_ETL_DAG' AIRFLOW_CTX_TASK_ID='insert_worldmeter' AIRFLOW_CTX_EXECUTION_DATE='2025-03-10T00:00:00+00:00' AIRFLOW_CTX_TRY_NUMBER='1' AIRFLOW_CTX_DAG_RUN_ID='scheduled__2025-03-10T00:00:00+00:00'
[2025-03-16T12:07:58.532+0000] {logging_mixin.py:151} INFO - Insertion des données dans worldmeter terminée.
[2025-03-16T12:08:01.787+0000] {python.py:194} INFO - Done. Returned value was: None
[2025-03-16T12:08:02.354+0000] {taskinstance.py:1400} INFO - Marking task as SUCCESS. dag_id=Airflow_ETL_DAG, task_id=insert_worldmeter, execution_date=20250310T000000, start_date=20250316T120538, end_date=20250316T120802
[2025-03-16T12:08:02.829+0000] {local_task_job_runner.py:228} INFO - Task exited with return code 0
[2025-03-16T12:08:03.071+0000] {taskinstance.py:2778} INFO - 0 downstream tasks scheduled from follow-on schedule check
